packages/react-native/ReactCommon/yoga/yoga/node/LayoutableChildren.h

```diff
@@ -0,0 +1,150 @@
+/*
```

Comment:
Hey @j-piasecki this is super cool love that you are contributing here!! My one bit of feedback is can we write tests for this class by itself? Meaning not relying on e2e layout conformance to test this indirectly? That way we can test specific node structures and how we expect to iterate over them

Reply:
To clarify, that one would be on Yoga side https://github.com/facebook/yoga/pull/1726

But I don’t think that has to block merging this. Could be a separate PR.